m2.md

```diff
@@ -0,0 +1,373 @@
+(indicating that the mutant was caught). The default value is `true`, but it can be turned off if desired.
+
+## Case Study
+
+We ran the tool on a large Move project, splitting up the test into 3 runs, light, medium-only and heavy-only.
+
+| Mode        | Mutants Alive | Mutants Killed | Percentage Alive |
+| ----------- | ------------- | -------------- | ---------------- |
+| light       | 88            | 353            | 20 %             |
+| medium-only | 1155          | 1458           | 44 %             |
+| heavy-only  | 1088          | 3952           | 22 %             |
+
+### Examples per operator
+
+As there are in total over 2000 surviving mutants, we can not cover them all. A few examples are given per operator.
+
+#### binary_operator_swap
+
+This mutation operator swaps the arguments of binary operators. For example, the `a - b` expression can be replaced with the `b - a` expression.
+
+Nearly all of these are test gaps. There are a few false positives (swap of operands on `&&` or `||` operator), severity should be assessed on a case by case basis.
+
+The loop in the following example would likely have zero iterations, but probably unlikely to be introduced by accident, as it would immediately look 'wrong' to a developer.
+
+```move
+dex_accounts::bulk_cancel_client_and_place_order_to_subaccount: Alive mutant
+--- original
++++ modified
+@@ -508,7 +508,7 @@
+
+         let i = 0;
+         let len = place_prices.length();
+-        while (i < len) {
++        while (len < i) {
+             let price = place_prices[i];
+             let size = place_sizes[i];
+             let is_buy = place_is_buys[i];
+
+```
+
+Another example, here the tests should assert that the remaining size is correctly calculated:
+
+```move
+async_matching_engine::trigger_pending_twap: Alive mutant
+--- original
++++ modified
+@@ -840,7 +840,7 @@
+                 account,
+                 order_id,
+                 orig_size,
+-                remaining_size: twap_size - total_fill_size,
++                remaining_size: total_fill_size - twap_size,
+                 is_buy,
+                 is_reduce_only,
+                 twap_start_time_s,
+```
+
+#### delete_statement
+
+This operator deletes entire statements.
+
+In this example, tests should assert that this move to global storage has happened.
+
+```move
+position_tp_sl_tracker::register_market: Alive mutant
+--- original
++++ modified
+@@ -56,13 +56,7 @@
+     }
+
+     public(package) fun register_market(market_signer: &signer) {
+-        move_to(
+-            market_signer,
+-            PendingOrderTracker {
+-                price_move_up_index: new_default_big_ordered_map(),
+-                price_move_down_index: new_default_big_ordered_map()
+-            }
+-        )
++        {}
+     }
+
+     public(package) fun get_account_from_pending_request(
+
```

Comment:
I guess this is part of the original diff snippet, but we could remove it for better clarity in the blog post.
```suggestion
```